### DISCUSSION TITLE
Server-Side Form Validation and Return Errors Messages

### DISCUSSION THREAD

**Discussion by italanchan** [Budibase/budibase#8830] [Help]
Hi, 

We are studying Budibase how to retrieve the server-side data validation results and errors messages if available.  However, it seems the discussions have no one use it.

Current, we have REST API for UI submit, it will validate data with business logic server and return the error results (form based or fields based error messages).  Is there any example or sample for it?

**Comment by JayP718** [comment]
Hi I would look at stored procedures or the equivalent depending on what you are using as you data source. You can raise an error after using you business logic.
